tools/clang/test/SemaHLSL/derivatives-allowed-on-mesh-nodes.hlsl

```diff
@@ -0,0 +1,22 @@
+// REQUIRES: dxil-1-9
```

Comment:
Did you verify that this actually runs (like `dxil-1-9`) is recognized and works?  You can insert something that will cause an error and run it to make sure it fails if not.

Reply:
Yes, running hcttest file-check on this test produces an error when I introduce a syntax error.